index.html

```diff
@@ -500,8 +500,8 @@
         <dfn><a href=
         "https://heycam.github.io/webidl/#idl-ArrayBuffer">ArrayBuffer</a></dfn>,
         <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-ArrayBufferView">ArrayBufferView</a></dfn>,
-        and the following exception names are defined in [[!WEBIDL-1]]:
+        "https://heycam.github.io/webidl/#ArrayBufferView">ArrayBufferView</a></dfn>,
+        and the following exception names are defined in [[!WEBIDL-2]]:
```

Comment:
Can you clarify what you think is not required?

We should enumerate all `<dfn>`s the spec depends on (for completeness and for ReSpec cross-referencing to work), and by convention indicate the references from where they originate from. There's no one true way, so editor has some freedom whether to use a single bulleted list or break definitions per reference, like is done here.